Example/iOS Example/AppDelegate.swift

```diff
@@ -43,6 +43,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
                 return
         }
         PiwikTracker.configureSharedInstance(withSiteID: siteid, baseURL: baseurl)
+        PiwikTracker.shared?.logger?.minLevel = .info
```

Comment:
I know it is not purpose of this PR but,
I would really suggest staying away from singleton and shared instances.

A design goal would be to run multiple trackers instances in the same app.

Let the consumer of the framework create their own shared instance if they want.